app/src/main/java/mx/yellowme/youst/challenges/ListenToMeChallengeActivity.kt

```diff
@@ -1,21 +1,19 @@
-package mx.yellowme.youst.presentation.challenges
+package mx.yellowme.youst.challenges
 
 import mx.yellowme.youst.R
-import mx.yellowme.youst.common.SimpleRecyclerAdapter
-import mx.yellowme.youst.common.activities.BaseChallengeActivity
-import mx.yellowme.youst.common.activities.ChallengeWithListActivity
+import mx.yellowme.youst.core.domain.Challenge
+import mx.yellowme.youst.core.domain.PokemonType
+import mx.yellowme.youst.core.domain.PokemonTypeSelection
+import mx.yellowme.youst.core.hooks.recycler.SimpleRecyclerAdapter
 import mx.yellowme.youst.pokemon.types.PokemonTypeAdapter
 import mx.yellowme.youst.pokemon.types.PokemonTypeListViewHolder
-import mx.yellowme.youst.domain.Challenge
-import mx.yellowme.youst.domain.PokemonType
-import mx.yellowme.youst.domain.PokemonTypeSelection
 import java.util.*
 
-class ListenToMeChallengeActivity : ChallengeWithListActivity<PokemonTypeSelection, PokemonTypeListViewHolder>(),
+class ListenToMeChallengeActivity :
```

Comment:
What do you think if we use the same splitting convention in all files? 🤔  
we have this in a file above 
```
class CrazyListsChallengeActivity : ChallengeWithListActivity<Pokemon, PokemonListViewHolder>(),
	    BaseChallengeActivity.MenuActionsListener,
	    ItemListener<Pokemon> {
```
and this one here
```
class ListenToMeChallengeActivity :
            ChallengeWithListActivity<PokemonTypeSelection, PokemonTypeListViewHolder>(),
	    BaseChallengeActivity.MenuActionsListener {
```
But I don't know which one is better 😅